docs/source/api_ref_models.rst

```diff
@@ -2,105 +2,409 @@
 torchtune.models
 ================
 
-.. currentmodule:: torchtune.models
 
-llama3
+This page contains information on all model architectures available in the core torchtune library.
+
+Llama3
 ------
 
-All models from the `Llama3 family <https://llama.meta.com/llama3/>`_.
+.. currentmodule:: torchtune.models.llama3
+
+`Llama 3 <https://llama.meta.com/llama3/>`_ - released in April of 2024 - is an auto-regressive language model that uses an optimized transformer architecture.
```

Comment:
I kind of see what you mean about maintaining the blurbs now, we could be writing a lot of these

Reply:
The description is interesting, but we dont follow the same standard for the other models. I think that giving general info, e.g. paper link, size, config link may be good enough.

Reply:
Personally I would support adding a quick 1-2 sentences at the top for each model though. I think this helps give a quick idea of what the main points are, then can point to references (imo this version has about 25% too much info though, keep it short and sweet). Agree with @felipemello1's point.. whatever precedent we set we should follow it for every model.